include/can/core/messages.hpp

```diff
@@ -336,13 +336,32 @@ struct ReadPresenceSensingVoltageResponse
         -> bool = default;
 };
 
+struct PushToolsDetectedNotification
+    : BaseMessage<MessageId::tools_detected_notification> {
+    uint16_t z_motor;
```

Comment:
The python side of things appears to think these are `uint8`. We should fix that but also IMO these should just be `enum` elements, a new specialization of `int_to_bytes`/`bytes_to_int` can handle that just fine